CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
   -------------------------------------------------------------------
 ## [Unreleased]
+### Changed
+- In Postgres, improved the FAKE_UPDATE strategy performances
```

Comment:
Could you write a little more on this in the changelog? you gave a really good description in your PR about what you've changed to make the speedup and which users might see the biggest speedups (i.e. updating large tables)

Reply:
Thanks for the suggestion, it's perfect !
I also added that another benefit of this new approach is that it makes performances less dependent on the size of the seed table.